tailwind.config.ts

```
@@ -0,0 +1,137 @@
+				}
+			},
+			animation: {
+				'accordion-down': 'accordion-down 0.2s ease-out',
+				'accordion-up': 'accordion-up 0.2s ease-out',
+				'fade-in': 'fade-in 0.5s ease-out',
+				'fade-in-left': 'fade-in-left 0.5s ease-out',
+				'fade-in-right': 'fade-in-right 0.5s ease-out'
+			}
+		}
+	},
+	plugins: [require("tailwindcss-animate")],
```

ESLint

[error] A `require()` style import is forbidden.

(@typescript-eslint/no-require-imports)